packages/experimental/Menu/src/MenuItemCheckbox/MenuItemCheckbox.types.ts

```diff
@@ -27,21 +35,19 @@ export interface MenuItemCheckboxProps extends Omit<IWithPressableOptions<ViewPr
    */
   componentRef?: React.RefObject<IFocusable>;
 
-  /**
-   * If the menu item is a trigger for a submenu
-   */
-  hasSubmenu?: boolean;
-
   /**
    * A callback to call on button click event
    */
   onClick?: (e: InteractionEvent) => void;
 }
 
-export type MenuItemCheckboxState = IPressableHooks<MenuItemCheckboxProps & React.ComponentPropsWithRef<any>>;
+export interface MenuItemCheckboxState extends IPressableHooks<MenuItemCheckboxProps & React.ComponentPropsWithRef<any>> {
+  hasCheckmarks?: boolean;
+}
 
 export interface MenuItemCheckboxSlotProps {
   root: React.PropsWithRef<IViewProps>;
+  checkmark?: SvgProps | XmlProps;
```

Comment:
Dumb question: Why does this need to be a union type?

Reply:
The union type I coped from somewhere else. Should probably figure out if it needs to be a union.

We could make it an Icon slot instead but since it's just for the checkmark (icon is a separate slot) I think it's okay for it to be more limited to start. If people want other icon types to be supported we can change it to Icon type.

Reply:
I'm not sure that it should be union here. I don't remember why I made it union. Maybe I expected that bug with SVG icons would be fixed earlier. Anyway, I think now it's better to have here only XmlProps. I found this commit:
https://github.com/microsoft/fluentui-react-native/pull/969/commits/4112cbf8dba50d202e56fe821a5a3a258ff82a25

Reply:
`XmlProps` extends `SvgProps` so we don't need the union, just using `XmlProps` should be enough